src/import/v8cpuFormatter.ts

```diff
@@ -0,0 +1,61 @@
+import {CPUProfile, CPUProfileNode} from './chrome'
+
+export interface OldCPUProfileNode {
```

Comment:
I don't think this needs exporting, does it?
```suggestion
interface OldCPUProfileNode {
```